src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/SettingsComponentTest.kt

```diff
@@ -111,7 +126,7 @@ class SettingsComponentTest {
 
     @Order(5)
     @Test
-    @Video
+    // @Video
```

Comment:
Video not working on mac?

Reply:
In order to get the tests to work I had to comment out the @video annotation in the SettingsComponentTest. I am confused if it is necessary to have that annotation for the GitHub actions or not, because it is used inconsistently throughout the project ( SettingsComponentTest and QuickAcessParametersTest classes) - from the description of the pull request.

Reply:
I don't really know how to run pipeline on dispatch can someone do that for Mac? I can try to add videos back but it doesn't quite work locally for me.

Reply:
I run your pipeline on dispatch for Windows and it passed :)

Reply:
Good what about Mac?

Reply:
Unfortunatelly, we are still in the process of making the dispatch run on Mac/Linux :)